setup.py

```diff
@@ -40,7 +40,6 @@
         "matplotlib",
         "Pillow",
         "tensorflow>=2.2.0",
-        "future-fstrings",
```

Comment:
I think this was required to support an early python3 version (maybe 3.6?).
Isn't there a place in `setup.py` to declare the minimum supported python3 version?